BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -196,7 +205,15 @@ public bool TryGetEnvironmentVariable(string name, out string value)
         /// <returns>The machine name.</returns>
         public string GetMachineName()
         {
-            return Environment.GetEnvironmentVariable("COMPUTERNAME");
+            try
+            {
+                return Environment.GetEnvironmentVariable("COMPUTERNAME");
+            }
+            catch (Exception e)
+            {
+                CoreEventSource.Log.FailedToLoadEnvironmentVariables(e.ToString());
+                throw;
```

Comment:
We don't want these methods to throw exceptions. The EventSource log will create internal traces that we can monitor.

@cijothomas what is the correct default value to return in this instance?

Reply:
Just removed the throw just like the others

Reply:
But the `throw;` is still there.